CRM/Contribute/Form/Contribution/ThankYou.php

```diff
@@ -284,6 +283,28 @@ public function buildQuickForm() {
       $this->assign('friendURL', $url);
     }
 
+    $isPendingOutcome = TRUE;
```

Comment:
I just tested & it is fine for pay later. So, for non-paylater it should reflect the outcome. I think it's safer if any doubt to give the 'we don't know yet' message